### ISSUE TITLE
How to tweak the learning rate of CRF layer?

### ISSUE THREAD

**Issue by Hansyvea** [allanj/pytorch_neural_crf#26]

**Comment by allanj** [comment]
In short, you can have a two optimization group (when creating the optimizer), one is other parameters, another is the parameters in the CRF layer (i.e., transition)
